py-polars/polars/io/csv/functions.py

```diff
@@ -1048,6 +1049,10 @@ def scan_csv(
         from environment variables.
     retries
         Number of retries if accessing a cloud instance fails.
+    file_cache_ttl
```

Comment:
Can we declare the unit? Is it seconds?

Reply:
It is there already

(look down 1-2 lines 😀)

Reply:
Sorry! :D